### DISCUSSION TITLE
Pulling a bare GitLab project (without an associated repository)

### DISCUSSION THREAD

**Discussion by cookiepresence** [magit/forge#723] [Support] [answered]
I have a GitLab project that does not have an associated repository, is there a way to use Forge to pull that without having to setup a repository for it?

**Comment by tarsius** [comment, marked as answer]
That's not supported and it likely will be a long time until I find the time to fill in such gaps. I concentrate on features available on all forges. While I had a different plans originally, the focus is on Github. And once I find the time to improve support for other forges, it will be on *adding* support for more forges, not on supporting features only available on Gitlab.
